init.ts

```diff
@@ -287,11 +287,20 @@ await Deno.writeTextFile(
   ROUTES_API_JOKE_TS,
 );
 
-const TWIND_CONFIG_TS = `import { Options } from "$fresh/plugins/twind.ts";
+const TWIND_CONFIG_TS = `import { Options } from "$fresh/plugins/twindv1.ts";
+
+import { Preset } from "https://esm.sh/v131/@twind/core@1.1.3/core.d.ts";
+
+import presetTailwind from "https://esm.sh/@twind/preset-tailwind@1.1.4";
+
+interface OptionsWithPresets extends Options {
```

Comment:
Should this type be included as part of the plugin exports so every user doesn't have to define it themselves?

Reply:
I may think so. I provided this in the init to avoid messing with too much file but it would be better if it was provided directly from the plugin exports 👍🏼